core/transport/src/main/proto/grpc/meta_master.proto

```diff
@@ -244,6 +246,19 @@ message MasterHeartbeatPResponse {
   optional MetaCommand command = 1;
 }
 
+message UpdateConfigurationPRequest {
+  map<string, string> properties = 1;
+}
+
+message UpdateConfigurationPResponse {
+  enum UpdatePropertyPStatus {
+    SUCCESS = 0;
+    BANNED = 1;
+    Exception = 2;
+  }
```

Comment:
proto enum is hard to read and maintain.
how about just use boolean to indicate success or failure?

Reply:
Sure